### PR TITLE
PLAT-129512: Add unit tests for Popup

### PR DESCRIPTION
**Pull request by stanca-pop-lgp**
Enact-DCO-1.0-Signed-off-by: Stanca Pop stanca.pop@lgepartner.com

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
- added extra unit tests for agate/Popup

### Links
[//]: # (Related issues, references)
PLAT-129512

### Comments

### CODE REVIEW

**Review by daniel-stoian-lgp — APPROVED** [review]

File: Popup/tests/Popup-specs.js [hunk 1]
```diff
@@ -30,36 +44,82 @@ describe('Popup specs', () => {
...
-		const expected = false;
-		const actual = popup.prop('centered');
+			const expected = 'center';
+			const actual = popup.prop('position');
 
-		expect(actual).toBe(expected);
+			expect(actual).toBe(expected);
+		});
+
+		test('should not have center class', () => {
```
Comment (0x64) [inline Popup/tests/Popup-specs.js:83]: What about updating this to explain what is the key point here? If I understand correctly, this test case is to distinguish `position` prop's value 'center' and class 'center' by `centered` prop. Am I right? :)

Comment (stanca-pop-lgp) [reply Popup/tests/Popup-specs.js:83]: I did not explain it correctly, true. What I wanted to point out was that when it has position=center it has default styling, without adding an extra "center" class as it does when it has position=top.

Comment (stanca-pop-lgp) [reply Popup/tests/Popup-specs.js:83]: I think maybe we could just skip this test.

Comment (stanca-pop-lgp) [reply Popup/tests/Popup-specs.js:83]: I will delete this test for now and add it in this task: PLAT-129752: Sync Popup positioning with Sandstone.

**Pull request merged**